### ISSUE TITLE
How to increase accuracy?

### ISSUE THREAD

**Issue by rupshali21** [hoseong-kim/sae-pytorch#6]
How can I increase the accuracy? What are the parameters? Is there any way to add a training loop?
